src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scenes/SampleLogin.unity

```diff
@@ -2675,6 +2675,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 96894e49d9863ec4db1d93a5e56f19cb, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+  <sceneToLoad>k__BackingField: SampleMain
```

Comment:
we should add this as well to the SampleLogin - Web3Auth.unity file otherwise is blank by default
<img width="660" alt="Screenshot 2023-11-30 at 09 39 32" src="https://github.com/ChainSafe/web3.unity/assets/104365141/7b492091-d25d-444c-988d-3962a2964228">

Reply:
While we are here, I guess we can put "Scene To Load" instead of "scene To Load"

Reply:
good point